generic/tests/cfg/multi_queues_test.cfg

```diff
@@ -94,3 +94,7 @@
             nics += " nic2"
             vms += " vm2"
             image_snapshot = yes
+            check_irqbalance_cmd = "service irqbalance status"
```

Comment:
I think it is better provide the default value in the `multi_vms_nics.py`'s`params.get()`.
Eg. `params.get("check_irqbalance_cmd", "service irqbalance status")`

Reply:
Agree with @hereischen , but recommend use `systemctl` as default.

Reply:
Thanks, update it.